core/src/main/kotlin/org/evomaster/core/output/service/naming/NumberedTestCaseNamingStrategy.kt

```diff
@@ -0,0 +1,26 @@
+package org.evomaster.core.output.service.naming
```

Comment:
see previous comment. unless this class is a service bean injectable with Guice, shouldn't be uner `.service.` package